components/bpmn/bpmn-explorer-web/src/web/js/actions.js

```diff
@@ -1005,3 +1004,446 @@ function generateReport(){
+                                    taskId = "N/A";
+                                }
+                                if (taskDefKey == activityId && endTime !== null ) {
+                                    var state = "Completed";
+                                    break;
+                                } else if (taskDefKey == activityId && endTime == null ) {
+                                    var state = "Active";
+                                    var endTime = "N/A";
+                                    break;
+                                } else {
+                                    var state = "Not Started";
+                                    var startTime = "N/A";
+                                    var endTime = "N/A";
+                                    var activityInstanceId = "N/A";
+                                    var taskId = "N/A";
+                                }
+
+                            }
+                         
+                            DIV = DIV + "<tr><td>"+state+"</td><td style='word-wrap: break-word'>"+activityName+"</td><td>"+activityType+"</td><td>"+startTime+"</td><td>"+endTime+"</td><td>"+taskId+"</td><td>"+activityInstanceId+"</td></tr>";
+
+                        }
+                        DIV = DIV+"</tbody></table></div>"
+                        $("#auditLog").html(DIV);
+
+                    }
+                });
+            }
+    });
+}
+
+//Gets the details of the variables in a running/active process instance
+function getVariablesOfRunningProcessInstances(id){
+
+    var url = "/" + CONTEXT + "/send?req=/bpmn/runtime/process-instances/"+id+"/variables";
+
+    $.ajax({
+        type: 'GET',
+        contentType: "application/json",
+        url: httpUrl + url,
+        success: function (data) {
+
+            $("#variables").html("");
+            var variableInfo = JSON.parse(data);
+            if (variableInfo.restVariables.length == 0) {
+                var DIV = "<h3> No variables for this process instance </h3>";
+                $("#variables").html(DIV);
+            } else {
+                var DIV = "<div style='height:100%;overflow:auto;'><table id ='table1'><thead><td>Name</td><td>Type</td><td>Value</td><td>Scope</td></thead><tbody>"
+                for (var k = 0; k < variableInfo.restVariables.length; k++) {
+                    var name = variableInfo.restVariables[k].name;
+                    var type =variableInfo.restVariables[k].type;
+                    var value = variableInfo.restVariables[k].value;                    
+                    var scope = variableInfo.restVariables[k].variableScope;
+
+                    DIV = DIV + "<tr><td style='word-wrap: break-word'>"+name+"</td><td>"+type+"</td><td style='word-wrap: break-word'>"+value+"</td><td>"+scope+"</td></tr>";
+                }
+                DIV = DIV+"</tbody></table></div>"
+                $("#variables").html(DIV);
+            }
```

Comment:
we can remove the else part like by delaring 
var DIV = "<h3> No variables for this process instance </h3>";
as default. then change it if (variableInfo.restVariables.length > 0) 
and set  $("#variables").html(DIV); outside if-else block